src/@chakra-ui/styles.ts

```diff
@@ -57,7 +57,7 @@ const styles = {
       marginBottom: "0",
     },
     "p *:last-child": {
-      "margin-bottom": "0",
+      marginBottom: "0",
```

Comment:
Good catch. I think this was an error introduced in the fork update PR since this line shouldn't exist, it was [removed](https://github.com/ethereum/ethereum-org-website/blob/dev/src/%40chakra-ui/gatsby-plugin/styles.ts#L56) in upstream.

I'll create a new PR to update these files.